charts/zk-operator/templates/_helpers.tpl

```diff
@@ -0,0 +1,62 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "zerok-cli.name" -}}
```

Comment:
why does this have zerok-cli? - lets fix this

Reply:
@CHNG7

Reply:
done with these changes